app/models/department.rb

```diff
@@ -8,6 +8,10 @@ class Department < ActiveRecord::Base
   has_many :items
   has_many :locations
   has_and_belongs_to_many :notifications
+  
+  validates_presence_of :shortname
+  
+  scope :active, -> { where( status: "Active" ) }
 
 
   def self.managing_people
```

Comment:
Probably? Right now it wouldn't be possible to chain `managing_items` with `managing_departments` for example. And it's more idiomatic. But otherwise not much benefit I suppose.